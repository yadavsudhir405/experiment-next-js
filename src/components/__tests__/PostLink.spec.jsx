import React from "react"
import { render } from "@testing-library/react"

import { PostLink } from "../index";

describe('PostLink', () => {
    it('should match the snapshot', () => {
        const {container} = render(<PostLink id="12" title='dummy'/>);

        expect(container).toMatchSnapshot();
    });
});
