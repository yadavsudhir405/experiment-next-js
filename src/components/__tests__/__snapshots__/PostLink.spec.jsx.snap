// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostLink should match the snapshot 1`] = `
<div>
  <a
    href="/p/12"
  >
    dummy
  </a>
</div>
`;
